data/Luminometrics/compiler.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Nov 18 18:24:27 2016
+
+@author: Vivian Zhong
+"""
+import pandas as pd
+import os
+
+files = os.listdir()
+master_df = pd.DataFrame(columns = ["SampleID", "Well", "Replicate", "Well Type", "Date", "Luminescence", "Normalized to Negative", "Vlumins"])
+for file in files:
+    if file.endswith(".csv"):
```

Comment:
I think you might be missing one conditional - the "master_datafile.csv" file will also be one of the CSV files in the current directory.